howto/how-to-set-up-a-blaster.md

```diff
@@ -0,0 +1,228 @@
+	{ "blaster/E-11D",
+		"blaster/E-11D/tracks/EzraTheme.wav",
+		StylePtr<Layers<AudioFlicker<RotateColorsX<Variation,Blue>,RotateColorsX<Variation,Rgb<0,0,128>>>,BlastL<White>, InOutTrL<TrInstant,TrFade<300>,Pulsing<RotateColorsX<Variation,Blue>,RotateColorsX<Variation,Rgb<0,0,10>>,3000>>>>(),
+		StylePtr<Lockup<BlastFadeout<BlastFadeout<Black,AudioFlicker<Black,Red>,250,EFFECT_FIRE>,AudioFlicker<Black,Red>,1500,EFFECT_STUN>,AudioFlicker<Black,Red>>>(),
+		"E-11D"
+	}
+};
+
+BladeConfig blades[] = {
+ { 10000,
+   WS281XBladePtr<108, bladePin, Color8::GRB, PowerPINS<bladePowerPin2> >(),
+   SimpleBladePtr<CreeXPE2RedTemplate<1000>, NoLED, NoLED, NoLED, bladePowerPin4, -1, -1, -1>(),
+   CONFIGARRAY(presets) },
+};
+
+#endif
+
+#ifdef CONFIG_BUTTONS
+Button FireButton(BUTTON_FIRE, powerButtonPin, "fire");
+Button ModeButton(BUTTON_MODE_SELECT, auxPin, "modeselect");
+//Button PowerButton(BUTTON_POWER, aux2Pin, "power"); //A third button to power on/off your weapon
+//Button ClipButton(BUTTON_CLIP_DETECT, blade4Pin, "clip"); //Actually clip sensor. It should be closed when the clip is in, and open when the clip is removed. So you can use either latching as single button or momentary if it is a physical clip inserted sensor.
+//Button ReloadButton(BUTTON_RELOAD, blade3Pin, "reload"); //Dedicated button for reloading.
+
+#endif
+
+/*** OS 7+
+#ifdef CONFIG_STYLES
+using BasicRedBlastWithBlueStunAndAudioFlicker = StylePtr<Lockup<BlastFadeout<BlastFadeout<Black,AudioFlicker<Black,Red>,250,EFFECT_FIRE>,AudioFlicker<Black,Blue>,1500,EFFECT_STUN>,AudioFlicker<Black,Red>>>;
+// Add to preset as: StylePtr<BasicRedBlastWithBlueStunAndAudioFlicker>()
+#endif
+*/
+```
+
+## CONFIG_TOP Section
+In the `CONFIG_TOP` section the following defines are the specifics for defining your weapon. For the rest of the defines please consult the [CONFIG_TOP documentation](config/the-config_top-section.html).
+
+### ENABLE_BLASTER_AUTO
+This define determines if your weapon has the automatic fire mode enabled. If you want your weapon to be semi-automatic only, simply comment it out.
+```cpp
+#define ENABLE_BLASTER_AUTO
+```
+
+### BLASTER_SHOTS_UNTIL_EMPTY
+This is your weapon round capacity per "clip" (i.e. before needing to reload). If you want an infinite ammo weapon, simply comment it out.
+```cpp
+#define BLASTER_SHOTS_UNTIL_EMPTY 15
+```
+
+### BLASTER_JAM_PERCENTAGE
+Here we set the change of the weapon jamming per shot. The range is 0-100. Please note that if you comment it out, it will be replaced with a random chance per ignition.
+```cpp
+#define BLASTER_JAM_PERCENTAGE 1
+```
+
+## CONFIG_PROP Section
+This section is currently trivial as `blaster.h` is the only currently available prop for blaster use.
+
+## CONFIG_PRESETS Section
+Here we have really two different sets of definitions: your `Preset` which define your soundfont and styles used, and the `BladeConfig` which define your hardware.
```

Comment:
Seems kind of silly to tell people what the CONFIG_PRESETS section does. We have an entire page about that already. I think this should focus on the specifics for blasters, not general explanations.